lib/python/ship_callsign.py

```diff
@@ -1,42 +1,36 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
 """
-Reads the ship_callsign text file in the same directory and returns a
+Reads the ship_callsign from platform vocab and returns
```

Comment:
I totally agree with you, BUT :
* it is not possible to filter by vessel. There is no way AFAIK to know if a platform is a vessel or mooring easily
* the only reason why there is this ship_callsign.py file is because the previous version of this file is still used by many data-services SOOP script which look for specific callsign. So in order not to change the different scripts, the ship_callsign.py stayed here. Otherwise, it means fixing everything in this PR which could also be done